### ISSUE TITLE
React: Applying theme to FilerobotImageEditor Typescript issues

### ISSUE THREAD

**Issue by josec0609** [scaleflex/filerobot-image-editor#338]
According to the [docs](https://github.com/scaleflex/filerobot-image-editor#theme) there is supposed to be a theme prop available for use in applying custom styling.

`FilerobotImageEditor` type definition does not contain a `theme` prop to pass to component call. 
https://github.com/scaleflex/filerobot-image-editor/blob/master/packages/react-filerobot-image-editor/src/index.d.ts#L240 

How do you apply a theme to the image editor in react if the editor component does not contain a valid `theme` prop?

**Comment by AhmeeedMostafa** [comment]
@josec0609 thanks for reporting, we will consider it in the upcoming version.

**Comment by rainabozhi** [comment]
will the upcoming version allowing custom themes be released any time soon? is there a time frame for this? 
best regards and thanks for the awesome editor either way

**Comment by AhmeeedMostafa** [comment]
@rainabozhi it's released as v4.5.0

**Comment by jducro** [comment]
Typescript is now complaining if all properties of palette or typography are missing meaning you have to specify all 37 colors of the palette, use `ThemeOverride` override should be more convenient: https://github.com/scaleflex/filerobot-image-editor/commit/2fb733430d3a82e3ba6acef67373926c004adb32#r117261479

**Comment by jducro** [comment]
@AhmeeedMostafa sorry to be pushy, this new type is quite blocking for us.

**Comment by jducro** [comment]
@AhmeeedMostafa would it be possible to release that fix as a patch release? Thanks

**Comment by AhmeeedMostafa** [comment]
@jducro yes will do within the next few days, we are abit busy sorry for the delay
